compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java

```diff
@@ -783,6 +783,7 @@ public enum DiagnosticErrorCode implements DiagnosticCode {
     UNSUPPORTED_COMPUTED_RESOURCE_ACCESS_PATH_SEGMENT_TYPE("BCE4031", 
             "unsupported.computed.resource.access.path.segment.type"),
     UNSUPPORTED_RESOURCE_ACCESS_REST_SEGMENT_TYPE("BCE4032", "unsupported.resource.access.rest.segment.type"),
+    INVALID_CLIENT_RESOURCE_RETURN_TYPE("BCE4033", "invalid.client.resource.return.type")
```

Comment:
```suggestion
    INVALID_RESOURCE_METHOD_RETURN_TYPE("BCE4033", "invalid.resource.method.return.type")
```

Reply:
What Maryam suggested in the meeting
`invalid resource method return type: cannot contain client objects and/or function types`